src/styl/responsive/mobile.styl

```diff
@@ -19,6 +29,45 @@
   svg
     height: 1.7rem
 
+#mobile-header
+  position: fixed
+  width: 100%
+  z-index: 1000
+
+#menu-toggle
+  svg
+    width: 50%
```

Comment:
This looks much better at 100%

Reply:
Even with 100% on the SVG the hit area is pretty small

Reply:
relying on the width of an s-1 doesnt work. the toggle is way too big on mobile landscape

![screen shot 2017-08-31 at 11 56 37 am](https://user-images.githubusercontent.com/7727722/29935174-78d1db56-8e43-11e7-8435-81d61b74bbbc.png)